### PR TITLE
Classroom

### PR DESCRIPTION
**Pull request by YuriiCheban93**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<description>Demo project for Spring Boot</description>
 	<properties>
 		<sonar.organization>if-135-java</sonar.organization>
- 		 <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
```
Comment (Forfend) [inline pom.xml:18]: Why did you add it to git ignore?

**Review by Forfend — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (YuriiCheban93) [reply pom.xml:18]: I didn't add it to git ignore.
